src/main/java/org/reaktivity/nukleus/socks/internal/stream/SocksServerFactory.java

```diff
@@ -395,38 +396,16 @@ private void onSocksConnect(
                     newRouteId, newInitialId, newReplyId);
 
                 SocksAddressFW requestAddress = socksConnectRequest.address();
-                StringFW address = formatSocksAddress(requestAddress);
 
                 socksConnectStream.doApplicationBegin(newTarget,
-                    decodeTraceId,
-                    address,
-                    socksConnectRequest.port());
+                                                      decodeTraceId,
+                                                      requestAddress.domainName(),
```

Comment:
Pass entire `requestAddress` to avoid assuming `domainName` case.

Reply:
done